src/ldp/http/BasicTargetExtractor.ts

```diff
@@ -6,13 +6,16 @@ import { TargetExtractor } from './TargetExtractor';
 
 /**
  * Extracts an identifier from an incoming {@link HttpRequest}.
- * Simply takes the input URl without any parsing/cleaning.
+ * Uses URL library for basic parsing but input requires more extensive cleaning/parsing based on headers.
  */
```

Comment:
Add TODO for sanitization

Reply:
(I know we have https://github.com/solid/community-server/issues/22, but for security issues, it's best to leave a note in the code)